spec/rubocop/target_ruby_spec.rb

```diff
@@ -188,10 +188,25 @@
 
       context 'when file reads the required_ruby_version from another file' do
         it 'uses the default target ruby version' do
```

Comment:
(This is an existing spec that I only modified to have a heredoc without interpolation; it is otherwise unchanged.)